diff --git a/lua/core/mappings.lua b/lua/core/mappings.lua
index 408f7a9..6d18292 100644
--- a/lua/core/mappings.lua
+++ b/lua/core/mappings.lua
@@ -290,7 +290,7 @@ M.nvterm = {
       "toggle floating term",
     },
 
-    ["<A-h>"] = {
+    ["<C-`>"] = {
       function()
         require("nvterm.terminal").toggle "horizontal"
       end,
@@ -314,7 +314,7 @@ M.nvterm = {
       "toggle floating term",
     },
 
-    ["<A-h>"] = {
+    ["<C-`>"] = {
       function()
         require("nvterm.terminal").toggle "horizontal"
       end,
diff --git a/lua/plugins/configs/cmp.lua b/lua/plugins/configs/cmp.lua
index 3bbc557..caf177e 100644
--- a/lua/plugins/configs/cmp.lua
+++ b/lua/plugins/configs/cmp.lua
@@ -79,7 +79,7 @@ local options = {
       behavior = cmp.ConfirmBehavior.Replace,
       select = false,
     },
-    ["<Tab>"] = cmp.mapping(function(fallback)
+    ["<Down>"] = cmp.mapping(function(fallback)
       if cmp.visible() then
         cmp.select_next_item()
       elseif require("luasnip").expand_or_jumpable() then
@@ -91,7 +91,7 @@ local options = {
       "i",
       "s",
     }),
-    ["<S-Tab>"] = cmp.mapping(function(fallback)
+    ["<Up>"] = cmp.mapping(function(fallback)
       if cmp.visible() then
         cmp.select_prev_item()
       elseif require("luasnip").jumpable(-1) then
